canonical/vanilla-framework
# add dark theme support to notifications

## Done
- add dark theme support to notifications with examples
- add dark theme to body elements

## Fixes [list issues/bugs if needed]
- https://warthogs.atlassian.net/browse/WD-8572

## QA
- open [demo](https://vanilla-framework-4968.demos.haus/docs/examples/patterns/notifications/notifications-dark) for a default notification
- open [demo](https://vanilla-framework-4968.demos.haus/docs/examples/patterns/notifications/variants-dark) for variants
- change `body` class from `is-dark` to `is-light` or `is-paper`, and check if background of the page and text color change accordingly

## Screenshots
<img width="1437" alt="Screenshot 2024-02-02 at 12 48 40 PM" src="https://github.com/canonical/vanilla-framework/assets/90341644/6d8ab373-ea6b-4476-8486-3e94871b1402">
